### ISSUE TITLE
How to set up a clone diablo, the character back to the city and has been waiting in the room?

### ISSUE THREAD

**Issue by wcg928** [kolton/d2bot-with-kolbot#513]
How to set up a clone diablo, the character back to the city and has been waiting in the room?
Sor can't kill clone diablo independently

**Comment by wojte5379** [comment]
yeah m2 wanna know..  today i get 2x hottest ip but he die and leave game... I need know it is any scrpit to stop hittings mobs or whatelse ? And go back to city when sojs soled ?
